eq-author/src/components/CommentsPanel/index.js

```diff
@@ -211,10 +206,34 @@ const CommentsPanel = ({
   const [activeReplyId, setActiveReplyId] = useState("");
   const [replyRef, setReplyRef] = useState();
   const [scrollRef, setScrollRef] = useState();
+  const {
+    match: {
+      params: { pageId, introductionId, sectionId, confirmationId },
+    },
+    me: { id: myId },
+  } = props;
+
+  const parentId = pageId || sectionId || confirmationId || introductionId;
+  const id = introductionId;
+
+  let key;
+
+  if (pageId) {
+    key = "pageId";
+  } else if (sectionId) {
+    key = "sectionId";
+  } else if (confirmationId) {
+    key = "confirmationId";
+  } else if (introductionId) {
+    key = "introductionId";
+  }
```

Comment:
https://codepen.io/MattElcock/pen/KKpzJqR?editors=0012 Have a look at this and see what you think; it could be used to tidy up the else if block.

Reply:
had to tweak it a bit as there was other info in the params that were messing it up - but yeah - lovely solution - thanks!